javascript/javascript2/week1/danish_string.js

```diff
@@ -1,14 +1,21 @@
 const danishString = ['Jeg har en blå bil'];
 //notThisFunctionName(danishString); // returns {total: 1, å: 1}
 
-function findShortestWord(str) {
-    var words = str.split(' ');
-    var shortest = words.reduce((shortestWord, currentWord) => {
-        return currentWord.length < shortestWord.length ? currentWord : shortestWord;
-    }, words[0]);
-    return shortest;
+var position = getbirdposition(danishString);
+
+function getbirdposition(danishString) {
```

Comment:
I don't understand what is the purpose of this function ? It's not part of the homework requirement.